tests/queries/0_stateless/02882_formatQuery.reference

```diff
@@ -26,6 +26,9 @@ SELECT 1
 SELECT 1
 1
 1
+SELECT 1 FORMAT TSV
```

Comment:
Same, let's move all the tests to a different file or files, please

Reply:
Added a new test `00309_formats_case_insensitive.sql`.